### ISSUE TITLE
Yubikey code not requested if password not configured

### ISSUE THREAD

**Issue by valorl** [aripalo/vegas-credentials#25]
**Describe the bug**
When using a Yubikey OATH account that's not configured with a password, `assume` returns early and doesn't proceed to request a code from the key.

**System (please complete the following information):**
Tested on  0.12.6 and `main`

**To Reproduce**
1. Setup a Yubikey, but do NOT run `ykman oath access change` to configure a password as the docs recommend
2. Run `assume` with a correctly configured `--profile`

**Expected behavior**
Prompt appears and vegas-credentials requests an OATH code from the key, therefore after touching the key, a Yubikey TOTP code is input.

**Output**
Log ends in:
```
DEBUG: 2022/05/09 17:01:42 setup.go:125 Yubikey: device is available
DEBUG: 2022/05/09 17:01:42 setup.go:132 Yubikey: OATH application is password protected
```
**Additional context**
Setting up a password, or building locally with [yubikey.go#L91-L102](https://github.com/aripalo/vegas-credentials/blob/main/internal/yubikey/yubikey.go#L91-L102) commented out fixes the issue.

It seems like [y.GetPassword()](https://github.com/aripalo/vegas-credentials/blob/14ab82fe394adf1b5f1576dfeebe8e400247a46a/internal/yubikey/yubikey.go#L91) returns a non-nil error `Key not found`, hence terminating the `Code` function.

**Comment by valorl** [comment]
I haven't dove deep enough into the error to figure out where exactly the `Key not found` is coming from (perhaps all the way from badger?). If you have some pointers on how this should be fixed, I'd be happy to make a PR for it :)

**Comment by aripalo** [comment]
Hi, thanks for reporting the issue!

I'll try to take a look at this tomorrow.

**Comment by aripalo** [comment]
Fixed in version `0.13.6`.

Essentially the issue boiled down to me not having tested with non-password protected Yubikeys for some time 😅 Sorry about that!
